simple_history/admin.py

```diff
@@ -193,10 +195,14 @@ def history_form_view(self, request, object_id, version_id, extra_context=None):
         context.update(self.admin_site.each_context(request))
         context.update(extra_context or {})
         extra_kwargs = {}
-        return render(
+        return self.render_history_view(
             request, self.object_history_form_template, context, **extra_kwargs
         )
 
+    def render_history_view(self, *args, **kwargs):
```

Comment:
```suggestion
    def render_history_view(self, request, template, context, **kwargs):
```
Can you just add to the signature, so we know what needs to be passed to `render_history_view`?